### ISSUE TITLE
[Bug] LetsEncrypt fails for subdomain if * wildcard DNS 

### ISSUE THREAD

**Issue by mehargags** [hestiacp/hestiacp#2563]
### Describe the bug

Just to clarify, I'm NOT trying to get an LE * wildcard SSL...
I have my DNS externally on HE.NET and made a wildcard *.domain.com >> IP
I configured 4 subdomains on hestia inside same hestia user.
I'm using all CLI commands for hestia.

When I try to fetch a letsencrypt SSL for subdomains, only the first subdomain can fetch it successfully. When I try to fetch SSL for next subdomains, it throws this error:

`sed: -e expression #1, char 34: unterminated `s' command`

If I remove the first subdomain, the other subdomain will fetch successfully. So to conclude, only ONE subdomain gets SSL, it fails for the next subdomains.

### Tell us how to replicate the bug

as explained above, it seems the script is somewhat checking a stake hash value or previous SSL with same domain and is exiting with an error. I highlighted error in the screenshots, both with CLI and from WEBUI

### Which components are affected by this bug?

Let's Encrypt SSL

### Hestia Control Panel Version

 v1.5.11

### Operating system

Debian 11.3

### Log capture

```shell
cat error.log
2022-04-29 13:20:55 v-list-web-domain  'niv' 'wiki.domain.com' 'plain' [Error 3]
2022-04-29 13:20:55 v-list-web-domain  'niv' 'wiki.domain.com' 'plain' [Error 3]
2022-04-29 13:20:55 v-add-web-domain  'niv' 'wiki.domain.com' [Error 2]
2022-04-29 13:21:27 v-list-web-domain  'niv' 'betterdocs.domain.com' 'plain' [Error 3]
2022-04-29 13:21:27 v-list-web-domain  'niv' 'betterdocs.domain.com' 'plain' [Error 3]
2022-04-29 13:21:27 v-add-web-domain  'niv' 'betterdocs.domain.com' [Error 2]
2022-04-29 13:21:38 v-list-web-domain  'niv' 'kb.domain.com' 'plain' [Error 3]
2022-04-29 13:21:38 v-list-web-domain  'niv' 'kb.domain.com' 'plain' [Error 3]
2022-04-29 13:21:38 v-add-web-domain  'niv' 'kb.domain.com' [Error 2]
2022-04-29 13:59:34 v-list-web-domain  'niv' 'gagan.domain.com' 'plain' [Error 3]
2022-04-29 13:59:34 v-list-web-domain  'niv' 'gagan.domain.com' 'plain' [Error 3]
2022-04-29 13:59:34 v-add-web-domain  'niv' 'gagan.domain.com' [Error 2]
2022-04-29 14:11:33 v-list-web-domain  'niv' 'rjss.domain.com' 'plain' [Error 3]
2022-04-29 14:11:33 v-list-web-domain  'niv' 'rjss.domain.com' 'plain' [Error 3]
2022-04-29 14:11:33 v-add-web-domain  'niv' 'rjss.domain.com' '185.213.175.131' 'yes' '' 'jpeg,jpg,png,gif,bmp,ico,svg,tif,tiff,css,js,htm,html,ttf,otf,webp,woff,txt,csv,rtf,doc,docx,xls,xlsx,ppt,pptx,odf,odp,ods,odt,pdf,psd,ai,eot,eps,ps,zip,tar,tgz,gz,rar,bz2,7z,aac,m4a,mp3,mp4,ogg,wav,wma,3gp,avi,flv,m4v,mkv,mov,mpeg,mpg,wmv,exe,iso,dmg,swf' [Error 2]




cat system.log
2022-04-29 13:20:55 v-restart-service  'php7.4-fpm'
2022-04-29 13:20:55 v-restart-service  'php8.1-fpm'
2022-04-29 13:20:55 v-add-web-domain-backend  'niv' 'wiki.domain.com' 'default' ''
2022-04-29 13:20:56 v-restart-service  'apache2' ''
2022-04-29 13:20:56 v-restart-service  'nginx' ''
2022-04-29 13:20:55 v-add-web-domain  'niv' 'wiki.domain.com'
2022-04-29 13:21:28 v-restart-service  'php7.4-fpm'
2022-04-29 13:21:28 v-restart-service  'php8.1-fpm'
2022-04-29 13:21:28 v-add-web-domain-backend  'niv' 'betterdocs.domain.com' 'default' ''
2022-04-29 13:21:28 v-restart-service  'apache2' ''
2022-04-29 13:21:28 v-restart-service  'nginx' ''
2022-04-29 13:21:28 v-add-web-domain  'niv' 'betterdocs.domain.com'
2022-04-29 13:21:39 v-restart-service  'php7.4-fpm'
2022-04-29 13:21:39 v-restart-service  'php8.1-fpm'
2022-04-29 13:21:39 v-add-web-domain-backend  'niv' 'kb.domain.com' 'default' ''
2022-04-29 13:21:39 v-restart-service  'apache2' ''
2022-04-29 13:21:39 v-restart-service  'nginx' ''
2022-04-29 13:21:39 v-add-web-domain  'niv' 'kb.domain.com'
2022-04-29 13:22:44 v-add-letsencrypt-user  'niv'
2022-04-29 13:22:46 v-restart-service  'nginx' ''
2022-04-29 13:23:02 v-generate-ssl-cert  'wiki.domain.com' 'info@wiki.domain.com' 'US' 'California' 'San Francisco' 'Hestia' 'IT' ''
2022-04-29 13:23:05 v-add-web-domain-ssl  'niv' 'wiki.domain.com' '/tmp/tmp.iVqYLuLDD0' 'same' 'updatessl'
2022-04-29 13:23:05 v-add-letsencrypt-domain  'niv' 'wiki.domain.com'
2022-04-29 13:24:01 v-restart-service  'apache2' ''
2022-04-29 13:24:02 v-restart-service  'nginx' 'ssl'
2022-04-29 13:53:25 v-restart-service  'nginx' ''
2022-04-29 13:53:41 v-generate-ssl-cert  'betterdocs.domain.com' 'info@betterdocs.domain.com' 'US' 'California' 'San Francisco' 'Hestia' 'IT' ''
2022-04-29 13:53:44 v-add-web-domain-ssl  'niv' 'betterdocs.domain.com' '/tmp/tmp.SrXl94HVzA' 'same
same' 'updatessl'
2022-04-29 13:53:44 v-add-letsencrypt-domain  'niv' 'betterdocs.domain.com'
2022-04-29 13:54:01 v-restart-service  'apache2' ''
2022-04-29 13:54:02 v-restart-service  'nginx' 'ssl'
2022-04-29 13:54:19 v-restart-service  'nginx' ''
2022-04-29 13:54:35 v-generate-ssl-cert  'kb.domain.com' 'info@kb.domain.com' 'US' 'California' 'San Francisco' 'Hestia' 'IT' ''
2022-04-29 13:54:38 v-add-web-domain-ssl  'niv' 'kb.domain.com' '/tmp/tmp.Vdl8urEB7B' 'same
same
same' 'updatessl'
2022-04-29 13:54:38 v-add-letsencrypt-domain  'niv' 'kb.domain.com'
2022-04-29 13:56:01 v-restart-service  'a
![2022-04-29_18-42-18](https://user-images.githubusercontent.com/2650395/166138810-1f47bc0b-7761-43a8-932d-16cc485a660e.jpg)
pache2' ''
2022-04-29 13:56:02 v-restart-service  'nginx' 'ssl'
2022-04-29 13:59:34 v-restart-service  'php7.4-fpm'
2022-04-29 13:59:34 v-restart-service  'php8.1-fpm'
2022-04-29 13:59:34 v-add-web-domain-backend  'niv' 'gagan.domain.com' 'default' ''
2022-04-29 13:59:35 v-restart-service  'apache2' ''
2022-04-29 13:59:35 v-restart-service  'nginx' ''
2022-04-29 13:59:34 v-add-web-domain  'niv' 'gagan.domain.com'
2022-04-29 13:59:50 v-restart-service  'nginx' ''
2022-04-29 14:00:06 v-generate-ssl-cert  'gagan.domain.com' 'info@gagan.domain.com' 'US' 'California' 'San Francisco' 'Hestia' 'IT' ''
2022-04-29 14:00:09 v-add-web-domain-ssl  'niv' 'gagan.domain.com' '/tmp/tmp.Y1oW2DPKSV' 'same
same
same
same' 'updatessl'
2022-04-29 14:00:09 v-add-letsencrypt-domain  'niv' 'gagan.domain.com'
2022-04-29 14:02:01 v-restart-service  'apache2' ''
2022-04-29 14:02:02 v-restart-service  'nginx' 'ssl'
2022-04-29 14:11:33 v-restart-service  'php7.4-fpm'
2022-04-29 14:11:33 v-restart-service  'php8.1-fpm'
2022-04-29 14:11:33 v-add-web-domain-backend  'niv' 'rjss.domain.com' 'default' 'yes'
2022-04-29 14:11:34 v-restart-service  'apache2' ''
2022-04-29 14:11:34 v-restart-service  'nginx' ''
2022-04-29 14:11:33 v-add-web-domain  'niv' 'rjss.domain.com' '185.213.175.131' 'yes' '' 'jpeg,jpg,png,gif,bmp,ico,svg,tif,tiff,css,js,htm,html,ttf,otf,webp,woff,txt,csv,rtf,doc,docx,xls,xlsx,ppt,pptx,odf,odp,ods,odt,pdf,psd,ai,eot,eps,ps,zip,tar,tgz,gz,rar,bz2,7z,aac,m4a,mp3,mp4,ogg,wav,wma,3gp,avi,flv,m4v,mkv,mov,mpeg,mpg,wmv,exe,iso,dmg,swf'
2022-04-29 14:11:34 v-change-web-domain-tpl  'niv' 'rjss.domain.com' 'default' 'no'
2022-04-29 14:11:34 v-restart-service  'apache2' ''
2022-04-29 14:11:34 v-restart-service  'nginx' ''
2022-04-29 14:11:49 v-delete-fastcgi-cache  'niv' 'rjss.domain.com'
2022-04-29 14:12:11 v-delete-fastcgi-cache  'niv' 'rjss.domain.com'
```
![2022-04-29_18-47-11](https://user-images.githubusercontent.com/2650395/166138771-24c45d17-7a61-42cd-b007-ea28a128fcff.jpg)

**Comment by jaapmarcus** [comment]
You try to generate an ssl for:

domain.com
sub1.domain.com
sub2.domain.com

Or domain.com and alias for 
sub1.domain.com
sub2.domain.com

**Comment by jaapmarcus** [comment]
Haven't been able to replicate the issue on 1.6.0 Beta. Are you able to test the beta version on a test server?

**Comment by mehargags** [comment]
@jaapmarcus 
I will give a shot to 1.6 beta and let you know my observations.

I have domain.com already configured and LE SSL fetched. 
Then I made a *.domain.com DNS Wildcard (on external DNS).
Now when you create a domain sub1.domain.com, it SUCCESSFULLY 'gets' the LE SSL but any subsequent sub2. sub3. sub4. will FAIL to fetch LE SSL giving out this error

```sed: -e expression #1, char 34: unterminated s' command```

hope I was more clear?

**Comment by jaapmarcus** [comment]
@mehargags  Any news when you tried the beta version?

**Comment by mehargags** [comment]
> @mehargags Any news when you tried the beta version?

@jaapmarcus sorry been on a short travel, will try and let you know soon.

Would you pls write how I can install beta or specific version?  is the `--beta` parameter for install working or I have to switch repo?

**Comment by jaapmarcus** [comment]
See:

https://forum.hestiacp.com/t/beta-hestia-1-6-0-help-wanted/6163

**Comment by jaapmarcus** [comment]
@mehargags  any news regarding this issue?

**Comment by jaapmarcus** [comment]
I will close the issue it should be fixed with the 1.6.0 release

**Comment by jaapmarcus** [comment]
Had the same experience with hestiacp.com domain last night:

```
root@web02:~# v-add-letsencrypt-domain hestia approval.hestiacp.com
sed: -e expression #1, char 34: unterminated `s' command
root@web02:~# v-add-letsencrypt-domain hestia approval.hestiacp.com
Error: invalid restart format :: same
same
same
same
same
same
same
same
Error: apache2 restart failed
root@web02:~# systemctl restart
```
